File: URI_1010_simple_calculate.js
const lines = ["12 1 5.30", "16 2 5.10"];

let prod1 = lines[0].split(" ");
let prod2 = lines[1].split(" ");

let valueToPay = Number(prod1[1]) * Number(prod1[2])
+ Number(prod2[1]) * Number(prod2[2]);

console.log("VALOR A PAGAR: R$ " + valueToPay.toFixed(2));

File: URI_1010_simple_calculate.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class URI_1010_simple_calculate {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		String[] prod1 = scanner.nextLine().split(" ");
		String[] prod2 = scanner.nextLine().split(" ");

		double valueToPay = Double.valueOf(prod1[1]) * Double.valueOf(prod1[2])
				+ Double.valueOf(prod2[1]) * Double.valueOf(prod2[2]);

		String formattedValueToPay = new DecimalFormat("0.00").format(valueToPay);
		System.out.println("VALOR A PAGAR: R$ " + formattedValueToPay);

		scanner.close();
	}
}
